src/options/imports/index_ops.js

```diff
@@ -0,0 +1,56 @@
+import index from 'src/search/search-index'
+
+const db = index.db._db
+
+async function downloadChunk(data, counter) {
+    chrome.downloads.download({
+        url: URL.createObjectURL(
+            new Blob([JSON.stringify(data)], { type: 'text/plain' }),
+        ),
+        filename: 'worldbrain_index_'.concat(counter, '.json'),
+    })
+}
+
+async function grabDBChunk(startAfter = '', limit = 1000) {
+    const data = []
+
+    return new Promise((resolve, reject) =>
+        db
+            .createReadStream({ gt: startAfter, limit })
+            .on('data', datum => data.push(datum))
+            .on('error', reject)
+            .on('end', () => resolve(data)),
+    )
+}
+
+/**
+ * @param {int} chunkSize Length of the chunk for piecewise export 
```

Comment:
No `int` type in JS. Generally for JSDoc just put `number` and specify expected range in description, as I don't think a lot of tooling would understand `int`.